Sample Applications/Win11ThemeGallery/Win11ThemeGallery.csproj

```diff
@@ -2,10 +2,13 @@
 
 	<PropertyGroup>
 		<OutputType>WinExe</OutputType>
-		<TargetFramework>net9.0-windows</TargetFramework>
+		<TargetFramework>net9.0-windows10.0.19041.0</TargetFramework>
```

Comment:
FYI, we're bringing a fix to WinAppSDK 1.5 that should make this change unnecessary, it may be released this week.  Once it's ready, let's use it and see if we can change this back to "net9.0-windows", etc.

Reply:
I tried this out locally, it looks like we'll need to keep this like you have it for now :(